File: Java/Faculdade/Atividade01/src/atividade01/Exercicio01.java
package atividade01;
import javax.swing.*;
public class Exercicio01 extends Animal{
    public static void main (String [] args){
        double valor=0;
        Animal a = new Animal();
        String animal = " ";
        String gato = "gato";
        String cachorro = "cachorro";
        String gatoecachorro = "gato e cachorro";
        animal = JOptionPane.showInputDialog(null,"Escolha um animal ou os dois:");
        if(animal.equals(gato)){
        animal = JOptionPane.showInputDialog(null,"Informe a quantidade de gramas da racao:");
        valor = Double.parseDouble(animal);
            a.Gato(0, valor);           
    }
        if(animal.equals(cachorro)){
        animal = JOptionPane.showInputDialog(null,"Informe a quantidade de gramas de racao:");
        valor = Double.parseDouble(animal);
            a.Cachorro(0, valor);
        }
        if(animal.equals(gatoecachorro)){
        animal = JOptionPane.showInputDialog(null,"Informe a quantidade de gramas de racao do gato:");
        valor = Double.parseDouble(animal);
        animal = JOptionPane.showInputDialog(null,"Informe a quantidade de gramas de racao do cachorro:");
        double valor2 = Double.parseDouble(animal);
            a.Gato(0, valor);
            a.Cachorro(0, valor2);
        }
    }
}

File: C++/Faculdade/Exercício/Segundo Semestre/Atividade00/Exercicio01.cpp
#include <stdio.h>

main(){
	
	int c, v[6];
	
	for(c=0;c<6;c++){
		printf("Informe o %d* valor do vetor: ",c+1);
		scanf("%d",&v[c]);
	}
	
	printf("\n");
	
	for(c=0;c<6;c++){
		printf("%d: %d \n",c+1,v[c]);
	}
	
	return 0;
}
